src/main/java/org/jvnet/hudson/test/HudsonPageCreator.java

```diff
@@ -23,40 +23,35 @@
  */
 package org.jvnet.hudson.test;
 
+import java.io.IOException;
+import java.util.Locale;
+
 import com.gargoylesoftware.htmlunit.DefaultPageCreator;
 import com.gargoylesoftware.htmlunit.Page;
+import com.gargoylesoftware.htmlunit.PageCreator;
 import com.gargoylesoftware.htmlunit.WebResponse;
 import com.gargoylesoftware.htmlunit.WebWindow;
-import com.gargoylesoftware.htmlunit.PageCreator;
-import org.apache.commons.lang3.StringUtils;
-
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.Locale;
 
 /**
  * {@link PageCreator} that understands JNLP file.
- * 
+ *
```

Comment:
Can you revert inessential changes such as whitespace and reformatting please? (You should be able to configure your IDE/editor to not modify lines of text unless and until a significant change is made on that line.)

Reply:
I can do yes. (If the rest of the changes are committed to reduce the overhead.)

Just a side note: This requires that I set up a new IDE configuration just for every Jenkins sub-project. Almost every other project I am contributing to defines a coding style that is also part of the project (i.e. in version control). Your approach by using no coding style at all makes contributing to Jenkins quite a burden. Normally, the IDE would be smart enough to handle those changes as you like them. In your case I need to manually revert such changes a smart IDE automatically takes care of.

Reply:
You do not need to manually revert changes if you do not make the changes in the first place, which you should be able to configure your IDE to do.

Reply:
Haha.